script.js

```diff
@@ -116,12 +117,52 @@ class Game {
     this.offsetX = offsetX;
     this.offsetY = offsetY;
     this.length = snakeLength;
+    this.keys = [];
     this.field = new Field(this.width, this.height, this.gridSize, this.offsetX, this.offsetY);
     this.snake = new Snake(this.field, this.length);
     this.stats = stats(width, height, gridSize, offsetX, offsetY);
     this.playButton = document.getElementsByClassName('play-button')[0];
 
     this.start = this.start.bind(this);
+    this.handleKeyPress = this.handleKeyPress.bind(this);
```

Comment:
You should bind all your methods in the constructor. This applies to both `start` and `handleKeyPress`.

Reply:
Ignore me.